src/messageComposer/middleware/textComposer/applyCommandSettings.ts

```diff
@@ -0,0 +1,55 @@
+import type { Middleware } from '../../../middleware';
+import type { TextComposerMiddlewareExecutorState } from './TextComposerMiddlewareExecutor';
+import type { CommandSuggestion } from './types';
+
+export type ApplyCommandSettingsMiddleware = Middleware<
+  TextComposerMiddlewareExecutorState<CommandSuggestion>,
+  'onChange' | 'onSuggestionItemSelect'
+>;
+
+export const createApplyCommandSettingsMiddleware =
```

Comment:
I would probably be looking for another name. It is not clear to me, what command settings mean in the name `createApplyCommandSettingsMiddleware`. The middleware is just removing string that corresponds to the trigger and command name.

Reply:
added new name